sidecar/codec_test.go

```diff
@@ -0,0 +1,68 @@
+package sidecar
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	hardcodedTicket = "sidecarAAQgHBgUEAwIBAAIBYwMBAgp5CwgAAAAAAAADCQwIAA" +
```

Comment:
😀

Reply:
base64 is kinda ugly imo, but hey, it encodes lol

Reply:
also we can also change it later if really needed

Reply:
Yeah, it's also not optimal because of the special characters (that's why I chose URL safe encoding here). Which encoding did you have in mind? bech32? I chose base64 for two reasons: A bit shorter than bech32 (double the alphabet size) and because PSBT uses the same (trying to convey the idea that this is similar in its concept as a transport format).